src/scripts/generate-config.sh

```diff
@@ -20,7 +20,7 @@ if [ "$tf" == true ]; then
     git clone --depth 1 --no-checkout git@github.com:FigurePOS/circle-ci-node-ecs-orb.git
     cd circle-ci-node-ecs-orb || exit
     git sparse-checkout set workflows
-    git checkout master
+    git checkout v1.9.0
```

Comment:
Tohle se asi bude chtít ještě doladit, už vidím jak se to bude zapomínat updatovat. 🙂

Reply:
A proč to nemůže bejt z masteru?

Reply:
Protože tam potřebuju udělat tuhle úpravu co jsme před chvílí revertovali: https://github.com/FigurePOS/circle-ci-node-ecs-orb/pull/22/files a jakmile ji zamerguju, tak všechny aktuální workflow začnou failovat než se updatnou na novou verzi orbu (jak se ukázalo na tom emailingu).

Reply:
A nebo já nevím 🤔 vrátím to zpět a rychle poupdatuju ty orby v servisách? Jak se to usadí a doladí poslední problémy, tak už by se to workflow nemělo měnit.

Reply:
já bych to radši udělal v klidu všechno :D

Reply:
No, mergnu tohle a když tak změním checkout zpátky na master později. 😄